.github/workflows/fuzzer.yml

```diff
@@ -0,0 +1,47 @@
+name: Fuzzer
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+jobs:
+  fuzzer:
+    name: Fuzzer
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Configure git
+        run: git config --global core.autocrlf false # required on Windows
+
+      - uses: actions/checkout@v4
+        with:
+          repository: SiaFoundation/fuzz
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go-version }}
+
+      - name: Build
+        run: go get go.sia.tech/core@none && go get go.sia.tech/coreutils@master && go mod tidy && go build -buildvcs=false .
```

Comment:
`@master` will probably always work, but we've noticed annoying caching in other workflows. Probably better to explicitly grab the commit hash you are wanting to test. Should be able to use `@$GITHUB_SHA`.